comms/src/transports/tcp.rs

```diff
@@ -81,9 +79,10 @@ impl TcpTransport {
 
     /// Apply socket options to `TcpStream`.
     fn configure(&self, socket: &TcpStream) -> io::Result<()> {
-        if let Some(keepalive) = self.keepalive {
-            socket.set_keepalive(keepalive)?;
-        }
+        // https://github.com/rust-lang/rust/issues/69774
+        // if let Some(keepalive) = self.keepalive {
+        //     socket.set_keepalive(keepalive)?;
+        // }
```

Comment:
remove?

Reply:
This (and all the others in this file) could come back - tokio doesn't support these socket options at the moment. I'd rather leave them in